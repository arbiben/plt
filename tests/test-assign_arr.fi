$tests filling struct arrays only partially and printing their contents/default values$

int main(){

arr int 6 a?
arr str 20 b?
b = ["not", "filled", "arrays :)"]?
printstring(b@20)?
a = [1, 2, 3]?
print(a@1)?
printstring(" ")?
print(a@5)?
printstring(b@2)?

}
