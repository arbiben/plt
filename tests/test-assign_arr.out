(null)2 0arrays :)